crt_portal/cts_forms/views.py

```diff
@@ -14,13 +14,15 @@
 from .models import Report, ProtectedClass
 from .model_variables import PROTECTED_CLASS_CODES
 from .page_through import pagination
-
+from .filters import ReportFilter
 
 SORT_DESC_CHAR = '-'
 
 
 @login_required
 def IndexView(request):
+    report_filter = ReportFilter(Report, request.GET.dict())
```

Comment:
So one thing that we may want to plan ahead on - `request.GET.dict()` will only return the last peram if it is a repeated peram. So if we want to search for two sections or extend the code to search for two or more things, in the future it won't work.

If we use something like `request.GET.getlist('assigned_section')` that will give us a list for the filter. (We could probably incorporate that part of the filter code in a more elegant way.)

I'll defer to you if you want to tackle that now, but I am fine merging as is if you don't want to worry about that yet.

Reply:
Oh that is a great point, I guess I was sort of hoping django would group everything into a bucket for you automatically 😄 . I'll just make that change now.